components/HomeHeader.vue

```diff
@@ -1,24 +1,24 @@
 <template>
   <swiper
-    v-if="items.length > 0 && !$vuetify.breakpoint.mobile"
+    v-if="items.length > 0 && !$vuetify.breakpoint.mobile && loaded"
     class="swiper"
     :options="swiperOptions"
   >
     <swiper-slide v-for="item in items" :key="item.Id">
-      <div
+      <blurhash-image
         class="slide-backdrop"
-        :style="{
-          backgroundImage: `url('${getBackdrop(item)}')`
-        }"
+        :item="getRelatedItem(item)"
+        :type="'Backdrop'"
       />
       <div class="slide-backdrop-overlay" />
       <div class="slide-content">
         <v-container class="mx-10 mt-5">
           <v-row>
             <v-col cols="5">
-              <v-img
+              <blurhash-image
```

Comment:
Mentioned in my logo PR, but logos shouldn't use blurhash, since they have transparency.